kibana/tests/kibana_test.py

```diff
@@ -629,3 +629,17 @@ def test_adding_loadBalancerIP():
     r = helm_template(config)
 
     assert r["service"][name]["spec"]["loadBalancerIP"] == "12.5.11.79"
+
+def test_service_port_name():
+    r = helm_template("")
+
+    assert r["service"][name]["spec"]["ports"][0]["name"] == "http"
```

Comment:
Can you move this assertion to https://github.com/elastic/helm-charts/blob/master/kibana/tests/kibana_test.py#L12?